### ISSUE TITLE
First noteobok's review

### ISSUE THREAD

**Issue by MarcoMagl** [LuxProvide/PennyLane-GPU#1]
In the following, what is the arg "shots" ?

```
# Alternatively, you can give labels to your qubits
dev_unique_wires = qml.device('default.qubit', wires=['q1', 'q2'], shots = [5, 10, 1000])
```

In the output of `result = quantum_circuit()` what means the argument "requires_grad" used for?

You draw the circuit with:

```
print(qml.draw(quantum_circuit)())
```

but it would have been nice to have an explanation on the visualization of the gate that we can see.

In the section on Pauli Gates X, Y,Z I would have explained what is the bloch sphere. Something like:

The Bloch sphere is a way to visualize a qubit's state as a point on a sphere, where the north and south poles correspond to the classical states 0 and 1, and all other points represent superpositions of those two states. It’s a helpful way to understand the complex nature of quantum information in a simple, geometric form.

**Comment by ekieffer** [comment]
- Table of content on the right for the notebook
- https://bloch.kherb.io/ to make bloch sphere picture
- Add some pictures (for ex. use cases, applications)

**Comment by Wahid612** [comment]
@MarcoMagl Have a look, I made some modifications.

**Comment by MarcoMagl** [comment]
**Introduction**
IMHO, you should mention (even if it is obvious) that Pennylane used in this tutorial use the GPU hardware to simulate quantum hardware. I would add something like:
In this tutorial, we are not using actual quantum hardware; instead, we simulate qubits by leveraging accelerators, specifically GPUs, to efficiently mimic quantum systems.

**Using PennyLane**

As we discussed, I think it is interesting to discuss the 'shot' thing.

This is what I propose:

In the context of quantum computing, the term "shots" refers to the number of times a quantum circuit is run (or executed) to collect data from a quantum measurement. Since quantum measurements are probabilistic in nature, a single execution of a circuit (a "shot") does not give a definitive answer but rather a sample from the probability distribution of possible outcomes.
As you can see, the shots is provided as an argument to the qml.device function with a list of 2 values. This is because one can run a different number of shots for the 2 different wires of the circuit. 

You can need a different number of shots in situations where you need more accuracy on one/some wires of your circuit.
More specifically, if certain qubits or parts of the circuit contribute more to the final result, you might want to allocate more shots (repeated measurements) to those areas to reduce uncertainty and improve accuracy. On the other hand, less critical parts of the circuit may not require as many shots, allowing you to save computational resources.

In most standard quantum circuits, having the same number of shots for all wires (qubits) is necessary to maintain coherence in measurements, particularly when the qubits are entangled or interacting. However, there may be specific scenarios where it could be interesting or useful to allocate different numbers of shots to different wires.

For instance, if on a 2 wire circuit I have a wire 1 which is highly susceptible to noise, while wire 2 is more stable, t may be beneficial to allocate more shots to wire 1 and fewer shots to wire 2. 


**QNode**

I would specify that this is a kind of terminology to talk about a quantum circuit. 


**Some more detail on cell 65**

This quantum circuit (which we can also call a qnode in the pennylane terminology) diagram shows our two-qubit system. The top wire (the qubit 0) undergoes a Hadamard gate (H) followed by a CNOT on the bottom wire (qubit 1). Both qubits are then measured (this is represented by the two symbols that you can see on the far right).


**Quantum circuits examples**

I would add something to explain that in this subsection, we give examples of the "building blocks" for standard quantum circuit.
